compiler/rustc_lint_defs/Cargo.toml

```diff
@@ -5,7 +5,6 @@ version = "0.0.0"
 edition = "2018"
 
 [dependencies]
-log = { package = "tracing", version = "0.1" }
```

Comment:
Dropping log/tracing seems like a not necessary step - it's unlikely to really help and the line here is pretty complex to readd or find as log macros get added or removed when refactoring.

Reply:
@Mark-Simulacrum I considered that, but I haven't found any signs that it was *ever* used in `rustc_lint_defs`, in the history since that crate was created.

Reply:
`rustc_lint_defs` also contains almost no code, only definitions for types and data.